### DISCUSSION TITLE
Нужно ли переводить все термины?

### DISCUSSION THREAD

**Discussion by ilyakmet** [rust-lang-ru/dictionary#25] [Ideas]
Проблема в том, что в словаре есть переводы терминов типа Trait, что явлеяется не совсем  удобным в работе, тем более, что в сообществе большинство терминов используют в их исходном виде (на английском).

Есть предложение не переводить те термины, которые уже известны сообществу в их оригинальном варианте. Критерий конечно звучит немного странно, но думаю читатели и переводчики понимают о чем идет речь. 

Если это необходимо, можно добавить столбец для того чтобы помечать такие термины.

**Comment by funkill** [comment]
А можешь ещё примеры привести (желательно предложениями с термином), ибо для меня "трейт" не показателен

**Comment by ava57r** [comment]
надо переводить если есть устоявшиеся термины.
Редкие и новые нужна экспертиза использования прежде чем переводить
